packages/back-end/src/models/FeatureRevisionModel.ts

```diff
@@ -50,7 +50,7 @@ featureRevisionSchema.index({ organization: 1, status: 1 });
 
 type FeatureRevisionDocument = mongoose.Document & FeatureRevisionInterface;
 
-const FeatureRevisionModel = mongoose.model<FeatureRevisionInterface>(
+export const FeatureRevisionModel = mongoose.model<FeatureRevisionInterface>(
```

Comment:
Don't export the model.  Instead, export a `countDocuments` method.

Reply:
whoops, fixed